node/sc/sub_bridge_handler.go

```diff
@@ -274,7 +274,14 @@ func (sbh *SubBridgeHandler) genUnsignedChainDataAnchoringTx(block *types.Block)
 		types.TxValueKeyAnchoredData: encodedCCTxData,
 	}
 
-	if tx, err := types.NewTransactionWithMap(types.TxTypeChainDataAnchoring, values); err != nil {
+	txType := types.TxTypeChainDataAnchoring
+
+	if feePayer := sbh.subbridge.bridgeAccounts.GetParentOperatorFeePayer(); feePayer != (common.Address{}) {
+		values[types.TxValueKeyFeePayer] = feePayer
+		txType = types.TxTypeFeeDelegatedChainDataAnchoring
+	}
+
```

Comment:
This code does not support ~WithRatio type.

Reply:
I didn't consider ~withRadio in this PR.
It will implementate laster.